src/Template/Layout/default.ctp

```diff
@@ -15,38 +9,20 @@ echo $this->fetch(
             <?= $this->fetch('title', read('App.title', env('HTTP_HOST'))) ?>
         </title>
 
+        <meta name="viewport" content="width=device-width, initial-scale=1">
         <meta name="author" content="<?= read('App.author') ?>">
+        <?= $this->Html->meta('icon') ?>
         <?= $this->fetch('meta') ?>
-
-
         <?= $this->AssetCompress->css('platform') ?>
-        <?= $this->fetch('css'); ?>
-        <!-- Le HTML5 shim, for IE6-8 support of HTML5 elements -->
-        <!--[if lt IE 9]>
-        <script src="//html5shim.googlecode.com/svn/trunk/html5.js"></script>
-        <script src="//oss.maxcdn.com/libs/respond.js/1.3.0/respond.min.js"></script>
-        <![endif]-->
+        <?= $this->fetch('css') ?>
+        <?= $this->fetch('headjs') ?>
```

Comment:
Another point would be to also add a controller level e.g., to sum it up:
* application
* plugin (if part of the request)
* controller (prefixed by plugin if part of the request)
* action (prefixed by plugin if part of the request, prefixed by controller)
each... top and bottom.
It is then to the implementer if to use a more global or more decoupled approach (more requests, smaller files) and where to strike the balance. It allows to defer most/all css/js after the dom is loaded.